docs/UserGuide.md

```diff
@@ -32,15 +32,15 @@ WedLog is a desktop app for wedding planning, optimized for use via a Command Li
     &emsp; 2.6.1 [Viewing help: `help`](#2-6-1-viewing-help-help)<br>
     &emsp; 2.6.2 [Undoing last action: `undo`](#2-6-2-undoing-last-action-undo)<br>
     &emsp; 2.6.3 [Redoing last action: `redo`](#2-6-3-redoing-last-action-redo)<br>
-    &emsp; 2.6.4 [Redoing last action: `clear`](#2-6-4-clearing-guest-and-vendor-lists-clear)<br>
+    &emsp; 2.6.4 [Clearing guest and vendor lists: `clear`](#2-6-4-clearing-guest-and-vendor-lists-clear)<br>
```

Comment:
Actually could we just say "Clearing all guests and vendors"? Feels a bit weird to say lists in UG, maybe it can be in the actual description of the command itself if needed.

Reply:
Resolved